docs/api/duckdb/multi-database-support.md

```diff
@@ -0,0 +1,157 @@
+# Connect DuckDB to other Databases: 
+
+Mosaic utilizes DuckDB, which [can supports multiple databases](https://duckdb.org/2024/01/26/multi-database-support-in-duckdb.html) such as MySQL, PostgreSQL, and SQLite, by using [DuckDB extensions](https://duckdb.org/docs/extensions/overview.html). This capability allows data to be seamlessly read into DuckDB and transferred between systems, enhancing flexibility and interoperability in data management.
+
+## Examples:
+### Connecting to `PostgreSQL`:
+
+``` js
+import { vg, clear } from "http://localhost:5173/dev/setup.js";
```

Comment:
This import is only valid for the Mosaic dev setup. We should use a normal import (`from '@uwdata/vgplot';`) instead and we should not include `clear` here.